middleware.ts

```diff
@@ -0,0 +1,14 @@
+export { default } from 'next-auth/middleware'
+
+export const config = {
+  matcher: [
+    '/',
+    '/dashboard',
+    '/checkIn',
+    '/checkOut',
+    '/inventory',
+    '/history',
+    '/settings',
+    '/settings/:path*',
+  ],
```

Comment:
If the dashboard path got removed with Andrew's commit, does it still need to be a part of the middleware?

Reply:
Good catch!